src/Controllers/DatabaseController.js

```diff
@@ -515,7 +515,9 @@ DatabaseController.prototype.canAddField = function(schema, className, object, a
 // Returns a promise.
 DatabaseController.prototype.deleteEverything = function() {
   this.schemaPromise = null;
-  return this.adapter.deleteAllClasses();
+  return Promise.all([
+    this.adapter.deleteAllClasses(),
+    this.schemaCache.clear()]);
```

Comment:
this seems awkwardly whitespaced, can you put the closing `]);` on the next line?